ext/imap/php_imap.stub.php

```diff
@@ -5,251 +5,250 @@
+function imap_getmailboxes($imap, string $reference, string $pattern): array|false {}
 
-/** @param resource $stream_id */
-function imap_fetch_overview($stream_id, string $sequence, int $options = 0): array|false {}
+/** @param resource $imap */
+function imap_fetch_overview($imap, string $sequence, int $flags = 0): array|false {}
 
 function imap_alerts(): array|false {}
 
 function imap_errors(): array|false {}
 
 function imap_last_error(): string|false {}
 
-/** @param resource $stream_id */
-function imap_search($stream_id, string $criteria, int $options = SE_FREE, string $charset = ""): array|false {}
+/** @param resource $imap */
+function imap_search($imap, string $criteria, int $flags = SE_FREE, string $charset = ""): array|false {}
 
-function imap_utf7_decode(string $buf): string|false {}
+function imap_utf7_decode(string $string): string|false {}
 
-function imap_utf7_encode(string $buf): string {}
+function imap_utf7_encode(string $string): string {}
 
 #ifdef HAVE_IMAP_MUTF7
-function imap_utf8_to_mutf7(string $in): string|false {}
+function imap_utf8_to_mutf7(string $string): string|false {}
 
-function imap_mutf7_to_utf8(string $in): string|false {}
+function imap_mutf7_to_utf8(string $string): string|false {}
 #endif
 
-function imap_mime_header_decode(string $str): array|false {}
+function imap_mime_header_decode(string $string): array|false {}
 
-/** @param resource $stream_id */
-function imap_thread($stream_id, int $options = SE_FREE): array|false {}
+/** @param resource $imap */
+function imap_thread($imap, int $flags = SE_FREE): array|false {}
 
 function imap_timeout(int $timeout_type, int $timeout = -1): int|bool {}
 
 #if defined(HAVE_IMAP2000) || defined(HAVE_IMAP2001)
-/** @param resource $stream_id */
-function imap_get_quota($stream_id, string $qroot): array|false {}
+/** @param resource $imap */
+function imap_get_quota($imap, string $quota_root): array|false {}
 
-/** @param resource $stream_id */
-function imap_get_quotaroot($stream_id, string $mbox): array|false {}
+/** @param resource $imap */
+function imap_get_quotaroot($imap, string $mailbox): array|false {}
 
-/** @param resource $stream_id */
-function imap_set_quota($stream_id, string $qroot, int $mailbox_size): bool {}
+/** @param resource $imap */
+function imap_set_quota($imap, string $quota_root, int $mailbox_size): bool {}
 
-/** @param resource $stream_id */
-function imap_setacl($stream_id, string $mailbox, string $id, string $rights): bool {}
+/** @param resource $imap */
+function imap_setacl($imap, string $mailbox, string $user_id, string $rights): bool {}
 
-/** @param resource $stream_id */
-function imap_getacl($stream_id, string $mailbox): array|false {}
+/** @param resource $imap */
+function imap_getacl($imap, string $mailbox): array|false {}
 #endif
 
 function imap_mail(string $to, string $subject, string $message, ?string $additional_headers = null, ?string $cc = null, ?string $bcc = null, ?string $rpath = null): bool {}
```

Comment:
the last param should be something like `$return_path`?

Reply:
Indeed, it should!